guides/doc-Provisioning_Guide/topics/proc_building-a-discovery-image.adoc

```diff
@@ -0,0 +1,119 @@
+[[building-a-discovery-image]]
+= Building a Discovery Image
+
+The Discovery image is a minimal operating system based on {RHEL} that is PXE-booted on hosts to acquire initial hardware information and to check in to the Satellite Server. Discovered hosts keep running the Satellite Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process. The `foreman-discovery-image` package contains this image. You must install the package on the {SmartProxy} that provides TFTP services.
```

Comment:
The `foreman-discovery-image` package contains this image. < this is only valid for Satellite, upstream it's installed externally.